contour/templates/contour-deployment.yaml

```diff
@@ -0,0 +1,79 @@
+{{ $root := . }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "contour.fullname" $root }}
+  labels:
+    app: {{ template "contour.fullname" $root }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    heritage: {{ .Release.Service | quote }}
+    release: {{ .Release.Name | quote }}
+spec:
+  replicas: {{ .Values.deployment.contour.replicas }}
+  selector:
+    matchLabels:
+       app: {{ include "contour.fullname" $root }}
+       release: {{ .Release.Name }}
+  {{- with .Values.deployment.contour.strategy }}
+  strategy:
+{{ toYaml . | indent 4 }}
+  {{- end }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "contour.fullname" $root }}
+        release: {{ .Release.Name }}
+      name: {{ template "contour.fullname" $root }}
+      annotations:
+        prometheus.io/scrape: "true"
+        prometheus.io/port: "{{ .Values.deployment.contour.port.healthz }}"
```

Comment:
@cw-sakamoto 
If this annotation is required, should you add `prometheus ` to `requirements.yaml`?

Reply:
@cw-ozaki 
No, Prometheus is not required for contour. This annotation is `prometheus exporter` and there is no problem if only publish in Prometheus format. 
By publishing in Prometheus format, it becomes possible to collect metrics with datadog and other monitoring tools.